ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -972,6 +976,22 @@ router bgp {{ router_bgp.as }}
 {%             endif %}
       {{ network_cli }}
 {%         endfor %}
+{%         for redistribute_route in router_bgp.address_family_ipv6_multicast.redistribute_routes | arista.avd.natural_sort('source_protocol') %}
+{%             if redistribute_route.source_protocol is arista.avd.defined %}
+{%                 set redistribute_route_cli = "redistribute " ~ redistribute_route.source_protocol %}
+{%                 if redistribute_route.source_protocol == "bgp" %}
```

Comment:
bgp is not for ipv6 multicast address family. Even if we do it reflect to global config

Reply:
Corrected it.